File: mybatis-plus-sample-logic-delete/src/main/java/com/baomidou/mybatisplus/samples/logic/Application.java
package com.baomidou.mybatisplus.samples.logic;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@MapperScan("com.baomidou.mybatisplus.samples.logic.mapper")
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }
}

File: mybatis-plus-sample-kotlin/src/main/kotlin/com/baomidou/mybatisplus/samples/kotlin/Application.kt
package com.baomidou.mybatisplus.samples.kotlin

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication


/**
 * @author nieqiurong
 */
@SpringBootApplication
open class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
